src/modules/ui.js

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable no-use-before-define */
+
+import changeCompleteStatus from './status.js';
+import { getCurrentList, deleteTask } from './add_remove.js';
+
+const taskListContainer = document.querySelector('#taskListContainer');
+
+const clearDisplay = () => {
+  const currentDisplay = document.querySelectorAll('.task-details');
+  currentDisplay.forEach((taskDetail) => {
+    taskDetail.parentElement.remove();
+  });
+};
+
+const addCheckBoxListener = (taskId) => {
+  const taskCheckBox = document.getElementById(taskId).querySelector("input[type='checkbox']");
+
+  taskCheckBox.addEventListener('change', (e) => {
+    const targetBox = e.target;
+    targetBox.parentElement.classList.toggle('strikethrough');
+    changeCompleteStatus(getCurrentList(), targetBox.parentElement.id);
+  });
+};
+
+const loadTaskList = () => {
+  getCurrentList().forEach((task) => {
+    displayTask(task);
+  });
+};
+
+const displayTask = (task) => {
+  const taskHolder = document.createElement('li');
+  taskHolder.setAttribute('id', task.index);
+  taskHolder.classList.add = 'taskCard';
+
+  const taskCheck = document.createElement('INPUT');
+  taskCheck.setAttribute('type', 'checkbox');
+  taskCheck.classList.add = 'completeStatus';
+  taskCheck.checked = task.completed;
+  if (task.completed) {
+    taskHolder.classList.toggle('strikethrough');
+  }
+
+  const taskDescription = document.createElement('input');
+  taskDescription.classList.add('task-details');
+  taskDescription.setAttribute('readonly', 'readonly');
+  taskDescription.value = task.description;
+
+  const menuButton = document.createElement('span');
+  menuButton.classList.add('taskMenu');
+  menuButton.innerHTML = '&#8942;';
+
+  const thrashButton = document.createElement('span');
+  thrashButton.classList.add('trashButton', 'd-none');
+  thrashButton.innerHTML = '&#9986;';
```

Comment:
Another suggestion 😁 Using this approach is good and efficient when you are changing one node or two but as you can see, when you are modifying multiple elements it gets out of control(Avoid Repetition). Hint: Look into innerHTML or insertAdjacentHTML 🧐